### ISSUE TITLE
[Enhancement] Provide a cell collapse/expand

### ISSUE THREAD

**Issue by tirams** [HappenApps/Quiver#852]
Please allow a way to collapse a cell to its first line of text only and expand to be able to see the items in a note without or with the details. a bulk collapse all / expand all would also be great.

**Comment by n8gray** [comment]
I agree.  A lot of my notes contain a sequence of requests and responses.  The responses are often quite long.  Sometimes I need to see all that detail, but a lot of the time I just want to see the request/response sequence.  Being able to fold the cells down would be a big win.

Bonus points if cell folding state persists when you return to a note you've previously viewed.

**Comment by HarrySonghurst** [comment]
This would be a great feature as it would allow Quiver to be used as a sort of 'flashcard' app as well.

**Comment by neerajbadlani** [comment]
+1

**Comment by areese801** [comment]
+1

**Comment by sefzig** [comment]
+1
